hikari/applications.py

```diff
@@ -137,15 +137,18 @@ class OAuth2Scope(str, enums.Enum):
     """
 
     APPLICATIONS_COMMANDS = "applications.commands"
-    """Allows your application's (slash) commands to be used in a guild.
+    """Allows your application's application commands to be used in a guild.
```

Comment:
Might as well just only talk about an application's commands instead of application commands. 

Does it have to be your application anyways?

Reply:
Yes, it is only that applications commands it can modify:
![image](https://user-images.githubusercontent.com/29100934/166066757-1e3ed22b-e3c8-40f5-a08f-69ef93dc4a4a.png)

And that is a typo, will fix it. Thanks for pointing it out